### ISSUE TITLE
Running a ecological simulation results in an error

### ISSUE THREAD

**Issue by macbrayne** [cristal-smac/ipd#3]
Running the provided example or anything else as an ecological simulation fails:

## Python version:
```
Python 3.8.6 (tags/v3.8.6:db45529, Sep 23 2020, 15:52:53) [MSC v.1927 64 bit (AMD64)] on win32
```
I'm running this on Windows 10 64-bit if that helps.

## Error Message:
```python
>>> e.run()
C:\Users\macbr\Programming\ipd\src\ipd.py:206: RuntimeWarning: overflow encountered in long_scalars
  self.base * parent * self.scores[strat.name] / total
C:\Users\macbr\Programming\ipd\src\ipd.py:193: RuntimeWarning: overflow encountered in long_scalars
  self.scores[strat.name]
C:\Users\macbr\Programming\ipd\src\ipd.py:192: RuntimeWarning: overflow encountered in long_scalars
  total += (
C:\Users\macbr\Programming\ipd\src\ipd.py:206: RuntimeWarning: invalid value encountered in long_scalars
  self.base * parent * self.scores[strat.name] / total
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\macbr\Programming\GitHub\ipd\src\ipd.py", line 205, in run
    self.historic.at[self.generation + 1, strat.name] = math.floor(
ValueError: cannot convert float NaN to integer
```
or
```python
>>> e.run()
C:\Users\macbr\Programming\ipd\src\ipd.py:206: RuntimeWarning: overflow encountered in long_scalars
  self.base * parent * self.scores[strat.name] / total
C:\Users\macbr\Programming\ipd\src\ipd.py:193: RuntimeWarning: overflow encountered in long_scalars
  self.scores[strat.name]
C:\Users\macbr\Programming\ipd\src\ipd.py:192: RuntimeWarning: overflow encountered in long_scalars
  total += (
Warning : max_iter reached
```
at one point also:
```python
>>> e.run()
C:\Users\macbr\Programming\ipd\src\ipd.py:206: RuntimeWarning: overflow encountered in long_scalars
  self.base * parent * self.scores[strat.name] / total
Warning : max_iter reached
```
## DrawPlot() Results:
These are the drawPlot() results for the example in the README.md
![Figure_](https://user-images.githubusercontent.com/27809595/97727335-9594aa80-1ad0-11eb-8123-d8eb11183a13.png)

The affected readme for reference f2f766db7e97bc6a2f257aaf02811fcdaf5aaf6b

Unfortunately I don't have much experience with python, else I would try to fixing these issues myself.
Macbrayne

**Comment by macbrayne** [comment]
Everything's working now.

**Comment by macbrayne** [comment]
Fix: 63a623bb86935da17dbb42497c7fbde105c15c7f
